conf/messages.en

```diff
@@ -97,8 +112,5 @@ signedOut.title = For your security, we signed you out
 signedOut.heading = For your security, we signed you out
 signedOut.guidance = We did not save your answers.
 
-unauthorised.title = There is a problem
```

Comment:
don't delete these!!!

Reply:
This is a bad mistake. This is fixed now

Reply:
has this been fixed?

Reply:
yes. https://github.com/hmrc/trader-goods-profiles-frontend/blob/TGP-539/create-nirms-number-page/conf/messages.en